src/Favicon/Favicon.php

```diff
@@ -185,7 +185,10 @@ private function getFavicon($url, $checkDefault = true) {
 
         // See if it's specified in a link tag in domain url.
         if (!$favicon) {
-            $favicon = $this->getInPage($url);
+            $favicon = trim($this->getInPage($url));
+        }
+        if (substr($favicon, 0, 2) === '//') {
+            $favicon = 'https:' . $favicon;
```

Comment:
That may be true, but we can't know for sure. And it can change, for example if a popular CMS decides to use an URL with relative protocol for its favicon.